libs/docs/src/docs/no-version/quickstart-odo.md

```diff
@@ -0,0 +1,219 @@
+           component: runtime
+           workingDir: ${PROJECT_SOURCE}
+           group:
+             isDefault: true
+             kind: build
+    +  - id: run
+    +    exec:
+    +      commandLine: node app.js
+    +      component: runtime
+    +      workingDir: ${PROJECT_SOURCE}
+    +      group:
+    +        isDefault: true
+    +        kind: run
+    ```
+
+9. Now the devfile is ready to be used to run the application 
+
+    ```yaml {% title="Devfile for Hello World application" filename=".devfile.yaml" %}
+    schemaVersion: <version>
+    components:
+      - name: runtime
+        container:
+          image: node:18-alpine
+          endpoints:
+            - name: http-3000
+              targetPort: 3000
+    commands:
+      - id: install
+        exec:
+          commandLine: npm install
+          component: runtime
+          workingDir: ${PROJECT_SOURCE}
+          group:
+            isDefault: true
+            kind: build
+      - id: run
+        exec:
+          commandLine: node app.js
+          component: runtime
+          workingDir: ${PROJECT_SOURCE}
+          group:
+            isDefault: true
+            kind: run
+    ```
+
+10. With `odo`, with a configured `kubectl` or `oc` installed run `odo dev` and you should see the following output
+    
+    ``` {% title="odo dev output" %}
+     __
+    /  \__     Developing using the "helloworld-devfile" Devfile
+    \__/  \    Namespace: default
+    /  \__/    odo version: v3.9.0
+    \__/
+
+    ⚠  You are using "default" namespace, odo may not work as expected in the default namespace.
+    ⚠  You may set a new namespace by running `odo create namespace <name>`, or set an existing one by running `odo set namespace <name>`
+
+    ↪ Running on the cluster in Dev mode
+    •  Waiting for Kubernetes resources  ...
+    ⚠  Pod is Pending
+    ✓  Pod is Running
+    ✓  Syncing files into the container [401ms]
+    ✓  Building your application in container (command: install) [1s]
+    •  Executing the application (command: run)  ...
+    -  Forwarding from 127.0.0.1:20001 -> 3000
+
+
+    ↪ Dev mode
+    Status:
+    Watching for changes in the current directory /path/to/project
+
+    Keyboard Commands:
+    [Ctrl+c] - Exit and delete resources from the cluster
+        [p] - Manually apply local changes to the application on the cluster
+    ```
+
+11. The application port `3000` served in the cluster gets routed to your host on a different port (in this case `20001`). Run `curl http://localhost:20001` and you should see the following output
+
+    ``` {% title="Response content returned by curl" %}
+    Hello world!%
+    ```
+
+12. (Optional) Normally it is recommended to use `odo init` to start your project from a devfile registry stack, see [Developing with Node.JS](https://odo.dev/docs/user-guides/quickstart/nodejs#step-2-initializing-your-application-odo-init) and [Command Reference: odo init](https://odo.dev/docs/command-reference/init) for details
```

Comment:
This should be more visible. We expect most users won't ever start writing the devfile from scratch.

Maybe make it bullet no. 4? And expanding it with how `odo init` can be used.